pages/Identity/Overview.md

```diff
@@ -7,22 +7,319 @@ menu: Identity
+     * @param endBlock Block number considered to be the end of the delegate permissions
+     */
+    event DSNPRemoveDelegate(address delegate, uint64 endBlock);
+
+    /**
+     * @dev Add or change permissions for delegate 
+     * @param newDelegate Address to delegate new permissions to
+     * @param permission Permission level
+     * 
+     * MUST be called by owner or other delegate with permissions
+     * MUST consider newDelegate to be valid from the beginning to time
+     * MUST emit DSNPAddDelegate
+     */
+    function delegate(address newDelegate, Permission permission) external;
+
+    /**
+     * @dev Add or change permissions for delegate by EIP-712 signature
+     * @param r ECDSA Signature r value
+     * @param s ECDSA Signature s value
+     * @param v EIP-155 calculated Signature v value
+     * @param newDelegate Address to delegate new permissions to
+     * @param permission Permission level
+     * 
+     * MUST be signed by owner or other delegate with permissions (implementation specific)
+     * MUST consider newDelegate to be valid from the beginning to time
+     * MUST emit DSNPAddDelegate
+     */
+    function delegateBySignature(bytes32 r, bytes32 s, uint32 v, address newDelegate, Permission permission) external;
+
+    /**
+     * @dev Remove Delegate
+     * @param delegate Address to remove all permissions from
+     * @param endBlock Block number to consider the permissions terminated (MUST be > 0x0). 
+     * 
+     * MUST be called by the delegate, owner, or other delegate with permissions
+     * MUST store endBlock for response in isAuthorizedToAnnounce
+     * MUST emit DSNPRemoveDelegate
+     */
+    function delegateRemove(address delegate, uint64 endBlock) external;
+
+    /**
+     * @dev Remove Delegate By EIP-712 Signature
+     * @param delegate Address to remove all permissions from
+     * @param endBlock Block number to consider the permissions terminated (MUST be > 0x0).
+     * @param r ECDSA Signature r value
+     * @param s ECDSA Signature s value
+     * @param v EIP-155 calculated Signature v value
+     * 
+     * MUST be signed by the delegate, owner, or other delegate with permissions
+     * MUST store endBlock for response in isAuthorizedToAnnounce
+     * MUST emit DSNPRemoveDelegate
+     */
+    function delegateRemoveBySignature(bytes32 r, bytes32 s, uint32 v, address delegate, uint64 endBlock) external;
+
+    /**
+     * @dev Checks to see if address is authorized to announce messages
+     * @param addr Address that is used to test against erecover
+     * @param permission Level of permission check. See Permission for details
+     * @param blockNumber Check for authorization at a particular block number
+     * @return boolean 
+     * 
+     * @dev Return MAY change as deauthorization can revoke past messages
+     */
+    function isAuthorizedToAnnounce(address addr, Permission permission, uint64 blockNumber) external view returns (bool);
```

Comment:
Switch to `isAuthorizedTo` as the Announce part is implied or adjustable from the permission param.